src/lib.rs

```diff
@@ -777,6 +783,141 @@ impl Client {
         self.0.shutdown()
     }
 
+    /// Sets the x, y, width, and height properties of the current window. All values must be `>= 0`.
+    pub fn set_window_rect(&self, x: i32, y: i32, width: i32, height: i32) -> impl Future<Item = Self, Error = error::CmdError> + 'static {
+        use webdriver::common::Nullable;
+
+        debug_assert!(x >= 0);
+        debug_assert!(y >= 0);
+        debug_assert!(width >= 0);
+        debug_assert!(height >= 0);
```

Comment:
I think I'd prefer if these were errors rather than assertions.
Alternatively, make the arguments here `u32`s so that that can't happen, and the convert them to `i32` for the internal representation.

Reply:
Arguably we should move to [`failure`](https://github.com/withoutboats/failure) which would make some of this nicer to deal with, but I think for now the thing to do would be to add a variant to `CmdError` to signify invalid parameters. Might come in handy elsewhere too!